src/lsm/segmented_array.zig

```diff
@@ -1,6 +1,7 @@
 const std = @import("std");
 
 const assert = std.debug.assert;
+const maybe = stdx.maybe;
```

Comment:
dead code, but better leave a is, we'll use it as some point anyway!

Reply:
Oops! I was tinkering with some asserts but ended up not committing them, I guess I left that in.